README.md

```diff
@@ -20,6 +20,8 @@ Enter the value returned by “matlabroot” in the field named “MATLAB root
 #### Configuring “Automatic” Option
 This option finds tests written using the MATLAB unit testing framework and/or Simulink Test and runs them. If the code is organized using projects, it will locate all test files in the project that have been classified as "Test". If the code does not leverage projects or uses a MATLAB version prior to R2019a, the plugin will discover all tests in the current Jenkins workspace including subfolders. 
 
+If you are using source code management (SCM) system, like Git, then the job must include an appropriate SCM configuration to checkout the code before running the MATLAB plugin. If you are not using any SCM system to manage your code, then a previous build step should make sure the code is available in the workspace before running the MATLAB plugin.
```

Comment:
If you are using a source code management (SCM) system such as Git, then the job must include an appropriate SCM configuration to check out the code before running the MATLAB plugin. If you do not use any SCM system to manage your code, then an additional build step in need to ensure the code is available in the workspace before running the MATLAB plugin.

Reply:
updated with very small change 
If you are using a source code management (SCM) system such as Git, then the job must include an appropriate SCM configuration to check out the code before running the MATLAB plugin. If you do not use any SCM system to manage your code, then an additional build step is required to ensure the code is available in the workspace before running the MATLAB plugin.

Reply:
It is classified by test developer while writing the tests. he has option to either write tests by extending the MATLAlB unit framework or by following few prerequisites which will classifies the file as test file.

Reply:
Done.